app/components/search.tsx

```diff
@@ -236,6 +236,7 @@ const ShowMoreSuggestions = ({
   } else if (extraQuestions.length === 0) {
     return (
       <Dialog onClose={onClose}>
+        <div style={{minWidth: '30em'}}></div>
```

Comment:
this fixes the issue, but would it be better to have it in the css file?

Reply:
if `<Dialog className=...` works, can you add a new class to CSS?